gammapy/detect/test_statistics.py

```diff
@@ -16,7 +16,7 @@
 from astropy.nddata.utils import extract_array
 from astropy.io import fits
 
-from ..stats import cash
+from _test_statistics import _cash, __amplitude_bounds, _cash_sum, _f_cash_root
```

Comment:
This fails on Python 3:
https://travis-ci.org/gammapy/gammapy/jobs/57322817#L1785
I didn't look in detail, but maybe it's just that you need to use a relative import here?